bundles/org.openhab.binding.asuswrt/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="tapocontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
tapocontrol => assuswrt
(not sure if there are more tapocontrol strings, but please search them to be sure nothing is left.